### DISCUSSION TITLE
vs code

### DISCUSSION THREAD

**Discussion by ayusharma03** [community/community#134998] [Programming Help] [answered]
### Body

how do i push my code directly from my vs code to github repo

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by MrDracs** [comment, marked as answer]
Pretty sure you can just download and install git, then in your vs code, search and install the github extension. After which you will have a version control tab. Make sure you are logged into github from VS code's account settings. Also, Github Desktop makes it easier to merge PRs and review code commits

**Reply by ayusharma03** [reply]
Thanks!! helpful
